### DISCUSSION TITLE
Rust Library

### DISCUSSION THREAD

**Discussion by mbergwall2222** [taskforcesh/bullmq#2495] [Ideas]
Thinking of creating an open source Rust library for Bullmq.... Fully compatible with Taskforce/bull-board. 

Would any teams be interested in using this? I have alternatives so would only do this if others would benefit.

**Comment by manast** [comment]
I do not recommend it. Here what I do think is better: https://bullmq.io/news/240321/introducing-bullmq-proxy/
You could instead write a thin client library in Rust for bullmq-proxy, much simpler and future proof.
